packages/angular/projects/dev/tsconfig.app.json

```diff
@@ -2,7 +2,13 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../out-tsc/app",
-    "types": []
+    "types": [],
+    "paths": {
+      "@clr/angular": ["dist/clr-angular"],
+      "@clr/angular/*": ["dist/clr-angular/*"],
+      "@cds/angular": ["dist/angular"],
+      "@cds/angular/*": ["dist/angular/*"]
```

Comment:
Path mapping added to demo apps only. At the top level tsconfig it was causing excessive build times as it would cause a circular lookup for ngcc. Related issue https://github.com/angular/angular/issues/38832